Userland/Libraries/LibTest/TestCase.h

```diff
@@ -105,5 +105,5 @@ void set_suite_setup_function(Function<void()> setup);
     auto identifier = (value);                                        \
     if (::Test::current_test_result() == ::Test::TestResult::Overrun) \
         return;                                                       \
-    if (::Test::is_reporting_enabled())                                         \
+    if (::Test::is_reporting_enabled())                               \
```

Comment:
This does not add the `GEN` macro (as the commit message says), and the whitespace change should be squashed into the commit that originally made the mistake.